row_012_api.py

```
@@ -0,0 +1,369 @@
+            mimetype="application/json",
+        )
+
+    return Response(
+        json.dumps({"error": "unknown token", "version": __version__}),
+        status=400,
+        mimetype="application/json",
+    )
+
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", debug=True)
```

Ruff (0.13.2)

Possible binding to all interfaces

(S104)